bin/templates/project/dependencies.gradle

```diff
@@ -28,16 +28,25 @@ repositories {
 
 dependencies {
 	// Keep in sync with lib/src/main/project.properties
+<<<<<<< HEAD
 	compile 'com.android.support:appcompat-v7:23.2.1'
 	compile 'com.android.support:cardview-v7:23.2.1'
 	compile 'com.android.support:support-v4:23.2.1'
 	compile 'com.android.support:recyclerview-v7:23.2.1'
 	compile 'com.android.support:design:23.2.1'
 	compile 'com.google.android.gms:play-services-gcm:9.0.2'
+=======
+	compile 'com.android.support:appcompat-v7:27.1.0'
+	compile 'com.android.support:cardview-v7:27.1.0'
+	compile 'com.android.support:support-v4:27.1.0'
+	compile 'com.android.support:recyclerview-v7:27.1.0'
+	compile 'com.android.support:design:27.1.0'
+	compile 'com.google.android.gms:play-services-gcm:11.8.0'
+>>>>>>> 7.0.0-tmp
```

Comment:
Do you need to also update the SDK/buildTool version in build.gradle from 23 -> 27?
" 
	compileSdkVersion 23
	buildToolsVersion "23.0.2"
"

Usually, they need to be the same version as the android support libraries.